app/models/db/Metrics.scala

```diff
@@ -18,15 +18,15 @@ import util.Utils.processException
 case class Metric(
     id: String,
     originatingSystem: OriginatingSystem,
-    composerId: Option[String],
-    storyBundleId: Option[String],
-    commissioningDesk: Option[String],
-    userDesk: Option[String],
-    inWorkflow: Option[Boolean],
-    inNewspaper: Option[Boolean],
+    composerId: Option[String] = None,
+    storyBundleId: Option[String] = None,
+    commissioningDesk: Option[String] = None,
+    userDesk: Option[String] = None,
+    inWorkflow: Boolean = false,
+    inNewspaper: Option[Boolean] = Some(false),
```

Comment:
Could this also be `Boolean` rather than `Option[Boolean]`

Reply:
Yes, now that the evolution is out, I'm making `inNewspaper` and `roundTrip` booleans.